handler/transport/backend.go

```diff
@@ -409,8 +412,8 @@ func removeConnectionHeaders(h http.Header) {
 	}
 }
 
-func removeHopHeaders(header http.Header) {
-	for _, h := range hopHeaders {
+func RemoveHopHeaders(header http.Header) {
```

Comment:
Why this case change? `RemoveHopHeaders()` is only used here.

Reply:
Now referenced by proxy.go